src/index.js

```diff
@@ -0,0 +1,39 @@
+import { MongoClient } from 'mongodb';
+
+const uri = 'mongodb://localhost:27017';
+const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+async function getDB() {
+  try {
+    await client.connect();
+    return client.db('kaende');
```

Comment:
Consider having the uri and  db name stored within environment variables to allow for easier configuration management in different envs (dev ,staging ,prod)